core/src/main/scala/org/apache/spark/ui/UIUtils.scala

```diff
@@ -303,13 +300,10 @@ private[spark] object UIUtils extends Logging {
             <div class="brand">
               <a href={prependBaseUri("/")} class="brand">
                 <img src={prependBaseUri("/static/snappydata/SnappyData-Logo-230X50.png")} />
-                <!-- <span class="version">{org.apache.spark.SPARK_VERSION}</span> -->
+                {getProductUINameNode}
                 {getProductVersionNode}
               </a>
             </div>
-            <p class="navbar-text pull-right">
-              <strong title={appName}>{shortAppName}</strong> application UI
-            </p>
```

Comment:
Why are there two separate methods in UIUtils: simpleSparkPageWithTabs and simpleSparkPageWithTabs_2? It looks like only the latter is being used by SnappyData dashboard so why is the change required here too?